src/defaultV2-theme/blocks/sd-title.scss

```diff
@@ -29,11 +29,12 @@
   flex-grow: 1;
 
   .sd-title {
+    --default-font-size: var(--sjs-font-surveytitle-size, calc(2 * (var(--sjs-font-size, 16px))));
```

Comment:
May be it would be better to introduce a global css variable --sjs-font--default-surveytitle-size